.github/issue_label_bot.yaml

```diff
@@ -0,0 +1,4 @@
+label-alias:
```

Comment:
Could you add a comment please linking to mlbot so people know what this is for?

Reply:
Ok it’s ready